.github/wix/dapr.wxs

```diff
@@ -20,22 +20,38 @@
 
     <!-- Directory Structure -->
     <Directory Id='TARGETDIR' Name='SourceDir'>
-      <Directory Id='INSTALLDIR'>
+      <Directory Id='INSTALLDIR' Name = 'dapr'>
+      <!-- Default Installation Directory set to C:\dapr -->
 
         <!-- Dapr EXE Component -->
         <Component Id='DaprEXE_Comp' Guid='83363a24-2323-489c-b9b7-a3fd7f0dacd1'>
           <File Id='DaprEXE' Name='dapr.exe' DiskId='1' Source='$(var.DaprEXEDirectory)/dapr.exe' KeyPath='yes'>
           </File>
         </Component>
 
+        <!-- Update PATH Environment Variable -->
+        <Component Id='UpdatePATH_Comp' Guid='0198647e-bae8-47e6-b9ff-01881ee540ae'>
+        <CreateFolder />
```

Comment:
Is `CreateFolder` needed here?

Reply:
It's apparently required for this block to be executed.

Reply:
Yes